Controller/TutorialController.php

```diff
@@ -103,4 +103,31 @@ public function collectionsAction()
             'form' => $form->createView(),
         );
     }
+
+    /**
+     * @Route("/tutorial/send-mail")
+     * @Template()
+     * @return array
+     */
+    public function sendMailAction()
+    {
+        // get the message factory so we can create messages
+        $messageFactory = $this->get('framework.message_factory');
+
+        // create a simple message
+        $message = $messageFactory->createHtmlMessage(
+            'the subject',
+            '<p>foo bar</p>'
+        );
+
+        // set some extra properties, just like you would do with a normal \Swift_Message
+        $message->setTo(
+            $this->getParameter('mailer_default_to_email')
+        );
+
+        // send it
+        $this->get('mailer')->send($message);
+
+        return array();
```

Comment:
I would only send an email after clicking a button/submitting a form/..
Sending a mail everytime the page is loaded seems a little bit too much to me.

An example using a template would also be nice, because this is wat we'll do most times.